test/aws/rails/railtie_test.rb

```diff
@@ -47,6 +47,55 @@ module Rails
         end
       end
 
+      describe '.add_sqsd_listener_middleware' do
+        it 'adds middleware when AWS_PROCESS_BEANSTALK_WORKER_REQUESTS is set to true' do
+          ENV['AWS_PROCESS_BEANSTALK_WORKER_REQUESTS'] = 'True'
```

Comment:
Setting the ENV in a test could cause side effects, so if possible would be better to set in a before and unset in an after.  Probably wouldn't ever be an issue here since you're deleting it later in the test - but if the test fails/raises an exception before its deleted it will remain set.

Reply:
An RSpec `after` hook might work

Reply:
Good point on the exception - using an RSpec hook makes more sense.